http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/GceRuntimeMonitor.scala

```diff
@@ -206,8 +206,10 @@ class GceRuntimeMonitor[F[_]: Parallel](
             failedRuntime(
               monitorContext,
               runtimeAndRuntimeConfig,
-              RuntimeErrorDetails("Can't retrieve instance yet. Possibly runtime creation failed in Google",
-                                  shortMessage = Some("fail_to_create")),
+              RuntimeErrorDetails(
+                "Can't retrieve instance yet. Possibly runtime creation failed in Google due to `ZONE_RESOURCE_POOL_EXHAUSTED` error. Refer to https://support.terra.bio/hc/en-us/articles/4403307463067 for creating the VM in a different zone",
```

Comment:
I wonder if something like below may be a bit clearer:

```suggestion
                "Cloud compute resource was not found. Creation may have failed in Google with `ZONE_RESOURCE_POOL_EXHAUSTED` error. This can happen frequently when creating compute with GPUs. Please try again later or refer to https://support.terra.bio/hc/en-us/articles/4403307463067 for creating compute in a different zone.",
```

Reply:
I don't like `Cloud compute resource was not found` very much becuz `Not found` is way too overloaded while this is a very specific case

Reply:
Followed up on this [Slack thread](https://broadinstitute.slack.com/archives/GB1VAM14J/p1627308545084000?thread_ts=1627068720.070600&cid=GB1VAM14J)